docs/how-to-guides/isis-developer-guides/developing-mac-m1-arm-systems.md

```diff
@@ -89,36 +98,19 @@ Building the ARM version of ISIS is easier than the Intel version because there
 5. Configure ISIS
     * `conda env create -n IsisArmDev -f environment.yml`
```

Comment:
has this actually worked, I would assume there are missing dependencies not on ARM yet. e.g. Bullet.

Reply:
The conda environment seemed to work for me.  I have the side-by-side setup described [here](https://taylorreiter.github.io/2022-04-05-Managing-multiple-architecture-specific-installations-of-conda-on-apple-M1/), and I can make a Conda env/Build ISIS from the ARM side.  It doesn't always work perfectly, but I think it's possible now, hence why I took out the "not yet possible" earlier and changed it to "less reliable".